util/jsonCheck.js

```diff
@@ -1,10 +1,6 @@
 module.exports = (filename) => {
-  // Returns if filename is invalid
-  if (!filename) {
-    return;
-  }
   // Adds a .json if the file deosnt have a .json
-  if (!filename.includes('.json')) {
+  if (filename && !filename.includes('.json')) {
     return filename + '.json';
   }
   return filename;
```

Comment:
useless return

Reply:
whats a useless return

Reply:
the final one

Reply:
wait, nvm, I forgot it could already include '.json'